README.md

```diff
@@ -12,6 +12,17 @@
 
 =======
 
+### Project requirements
+
+- Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
+- Create a new game with the name of your choice by using the API.
+- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
+- Implement the form "Submit" button (sending data to the API).
+- Use `async` and `await` JavaScript features to consume the API.
+- No styling is required.
+
+
```

Comment:
[OPTIONAL]
- Keep your professionalism, provide a License at to your README file.

Reply:
Orale! That's a good idea. I will add it Thanks Carlos ! :D